### ISSUE TITLE
Support markdown in Slack

### ISSUE THREAD

**Issue by adityathebe** [containrrr/shoutrrr#386]
I don't see Query/Param Props to use markdown for Slack service. If I send markdown text anyway, it's just raw text that's sent.

**Comment by piksel** [comment]
Yes, this is due to shoutrrr using the `color` property of attachments, which isn't available in the `blocks` API. It's mostly for backwards compatibility though, so perhaps having a way to opt out of it to enable `mrkdwn` formatting could be added.

**Comment by piksel** [comment]
Actually, attachments can use `mrkdwn` by specifying the `mrkdwn_in` field, so this could still be added without breaking the existing functionality.
